website/src/website/data/playground-samples/interacting-with-the-editor/listening-to-mouse-events/sample.js

```diff
@@ -45,16 +45,16 @@ editor.changeViewZones(function (changeAccessor) {
 
 // Add a content widget (scrolls inline with text)
 var contentWidget = {
-	domNode: null,
+	domNode: (function () {
```

Comment:
`this.domNode` was recursively typed.
I instatiated eagerly because there is no need to delay instantiation.